requirements.txt

```diff
@@ -2,7 +2,7 @@ scipy
 numpy
 Jinja2
 pyyaml
-pyccl==2.1.0
+pyccl
```

Comment:
Actually 2.5.0 if one wants to use SSC. Added

Reply:
@carlosggarcia I think we should fix the `pyccl==2.5.0` in requirements.txt instead of leaving just `pyccl`? I have been explicitly fixing the version to prevent possible bugs in TJPCov from the upgrade of its dependencies

Reply:
@felipeaoli The problem were the picklings in the tests. Now, instead of having the pkl pregenerated, y generate them at running time and load them. That way there is no problem with the version.